BMM.Core/Translation/en/main.json

```diff
@@ -324,6 +326,10 @@
     "OptionAppVersionHeader": "App version",
     "OptionDeleteAccountHeader": "Delete Account",
     "OptionDeleteAccountText": "Request that your account is deleted.",
+    "OptionBibleStudyBadgeHeader": "Bible Study Notification badge",
```

Comment:
the translation for that was added in a previous commit, but I messed up a merge with crowdin, so that it was removed again. If we keep the ID crowdin should be able to restore if a language has already been translated.